src/App.tsx

```diff
@@ -27,13 +28,18 @@ interface AppProps {
   history: History<unknown>;
 }
 
+// Map with all the roles mapped to their landing pages
+const roleMap = new Map<Role,string>();
+roleMap.set(Role.ADMIN, "/private")
+roleMap.set(Role.RESEARCHER, "/researcher")
```

Comment:
🔧 No need to use a ES6 Map here, just use a plain old object

```suggestion
// Map with all the roles mapped to their landing pages
const roleMap = {
  [Role.ADMIN]: "/private",
  [Role.RESEARCHER]: "/researcher",
}
```